src/AutoRest.CSharp/Common/Output/Expressions/AzureExtensibleSnippets.cs

```diff
@@ -28,6 +28,7 @@ internal class AzureExtensibleSnippets : ExtensibleSnippets
 
         internal class AzureModelSnippets : ModelSnippets
```

Comment:
should these classes be private?

Reply:
Not sure right now. This would probably change in the future when we finalize the DI for Azure-specific types.